Translate pref

Vuradu opened the pull request:
### **Этот драфт используется для перевода интерфейса**

Нужно поработать над:

- [ ] Мусор (часть переведена машинно-коряво, часть названий не переведена, нужны советы и лор).
- [ ] Изменить "Вегетарианский ИРП" на "ИРП, Вегетарианский" и др. по типу с "ИРП, Меню Х".
- [ ] Пофиксить инициализацию ID карт.
- [ ] Разобраться с аурами. (лор, названия)
- [ ] Перепроверить постеры на корректный перевод.
- [ ] Изменить "бутылка для таблеток" на банку-баночку.
- [ ] Проверить переведённые в файле(не переведённые в игре) шкафы и ящики на предмет долбоебизма, по пути "/obj/structure/closet/wardrobe/ххх_sierra"
- [ ] Перепроверить всё, что связано с техами и переименовать "техническое обслуживание" в "техническое помещение"
- [ ] Перевести в карте все названия шлюзов и доп. объектов
- [ ] изменить "прожектор" на "мощная лампа". речь о настенных лампочках
- [ ] /obj/item/clothing/head/hardhat/damage_control/emergency - шлем УЧС(бюро по управлению чрезвычайными ситуациями). Уточнить лорную часть.
- [ ] Заменить СБ на ОЗА
- [ ] Пофиксить "маск адля дыхания", "аиолетовый ковёр", "медицинсКИЙ курьерская сумка и т.п."
- [ ] Пофиксить "большая урна" на "большой ящик"
- [ ] Проверить корректность сокращений и перевести их - code/obj/modules_files/clothing/suits/labcoast.dm (**лор**)
- [ ] Изменить все аббревиатуры на кириллицу. PCRC - ПЦРК, GCC - ГКК и т.д.
- [ ] Уточнить перевод аббревиатур mods\_rus\code\obj\modules_files\clothing\suits 
- [ ] Перевести кодекс code\modules\codex
- [ ] Фиксить цвет tan на бежевый.
- [ ] Фиксить "улучшенный шлем медицинс**КИЙ** скафандра "
- [ ] Изменить "золотое руно" на "жёлтое полотенце"
- [ ] Пофиксить "**м**паковка жевательных резинок"
- [ ] Разобраться с синглтонами. code\modules\crafting , code\modules\culture_descriptor, code\modules\emotes. Переменные находятся только в синглтоне.
- [ ] Перепроверить названия в mods\_rus\code\obj\modules_files\crafting_datum.dm
- [ ] Создать отдельные define для существительных семян. Сейчас семена переведены так: "пакет чили **seeds**". "seeds" прописаны в дефайнах - code\__defines\hydroponics.dm.
- [ ] Пофиксить "блю**З**спейс двигатель" 
- [ ] узнать, что такое "greytide worldwide"

Vuradu committed:
rename this word

Update augments.dm

Vuradu committed:
translate left leg

Vuradu committed:
Translate Adjust limbs

Перевёл настройку конечностей.
(вернул к исходным значением то, что не нужно было трогать)

Vuradu committed:
Translate organs

Перевёл настройку органов.

Vuradu committed:
Translane body settings

Перевёл настройку основной информации о персонаже.

Vuradu committed:
Translate equipment

Перевёл всё, что связано с одеждой(нижним бельём), и саму одежду

Vuradu committed:
Translate misc Physical staff

Перевёл всё остальное на вкладке Physical.

Vuradu committed:
Translate system buttons

Перевёл верхние кнопочки

YegorKandziuba committed:
Мод на руссификацию. Пожалуйста, прости меня бог

YegorKandziuba committed:
Revert "Translate system buttons"

This reverts commit 0069d4ec9e9f64fcebb580cff694673c85b47924.

YegorKandziuba committed:
Revert "Translate misc Physical staff"

This reverts commit 003ea0177d3dae7442204bd1e124175bd35f6c4e.

YegorKandziuba committed:
Revert "Translate equipment"

This reverts commit 2fa9d4e7b7d64469026785790bf45910d33ba42a.

YegorKandziuba committed:
Revert "Translane body settings"

This reverts commit 17d1f6e7597f8be4659da6a714ec6a671d5dafb6.

YegorKandziuba committed:
Revert "Translate organs"

This reverts commit e3b2f274c9bec5ff5c1ed366eede9ac62410b4a2.

YegorKandziuba committed:
Revert "Translate Adjust limbs"

This reverts commit d7a68f339b2a48ff8edc35b6f24b6492a69adf18.

YegorKandziuba committed:
Revert "translate left leg"

This reverts commit 72cfcb232a5b5c1ad15a2221fe4242e746056f04.

YegorKandziuba committed:
Revert "rename this word"

This reverts commit a7e9fa48a8c242753ecccdacc3ff36ba4388a2b7.

YegorKandziuba commented:
Аккуратно ревертнул действия Вураду, что бы тот делал перевод немного по  другому

YegorKandziuba committed:
Merge branch 'dev-sierra' into Translate_pref

Vuradu committed:
Translate some storage

Просто немного перевода

Vuradu committed:
Fix #include and missing "

пофиксил говно

Vuradu committed:
Translate storage staff

Вторая партия перевода хранилищ

Vuradu committed:
Final storage translate.

Перевёл всё из раздела контейнеров и замутил под них отдельную папку.

Vuradu committed:
Translate radio

Перевёл всё в папке радио

YegorKandziuba committed:
Переделка экзамайна и новые переменные

Vuradu committed:
Сhange 1st simbol in names

Поменял регистр имён

Vuradu committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into Translate_pref

Vuradu committed:
Merge remote-tracking branch 'upstream/dev-sierra' into Translate_pref

YegorKandziuba committed:
сру говно

YegorKandziuba committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into pr/1236

Vuradu committed:
More translate

Vuradu committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into Translate_pref

Vuradu committed:
fix shit

Vuradu committed:
Translate scanners

Забубенил сканеры и начал всякий стафф

Vuradu committed:
Translate all devices

Перевёл все девайсы

Vuradu committed:
Fix tranlation

немного фиксов

YegorKandziuba committed:
Су+Ка

Vuradu committed:
Translate robots

Vuradu committed:
Translate robot staff

Vuradu committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into Translate_pref

YegorKandziuba committed:
коммит говна

YegorKandziuba committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into pr/1236

SuhEugene commented on mods/_rus/_rus.dm:
```diff
@@ -0,0 +1,12 @@
+	var/name_rus
+	var/desc_rus
+
+/atom/Initialize(mapload, ...)
+	. = ..()
+	if(!name_rus) name_rus = name
+	if(!desc_rus) desc_rus = desc
```
Пиздец блять.
Это явно лишнее.

Лучше в коде использовать `name_rus || name`, наверное, чем залезать в инициализацию каждого атома

SuhEugene commented on mods/_rus/_rus.dme:
```diff
@@ -0,0 +1,23 @@
+#ifndef MODPACK_RUSSIAN_NAME
+#define MODPACK_RUSSIAN_NAME
```
Ты явно не скриптом модпак генерил

Если папка называется `_rus`, то и мод должен быть `MODPACK__RUS`

Либо нужно папку переимновать в `russian_name` как и dm/dme

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "Устройство для воспроизведения ритмичных звуков, в просторечии называемое 'бумбоксом'. Выполнен в ретро-стиле и довльно громоздкий."
+
+/obj/item/device/chameleon
+	name = "устройство 'Хамелеон'"
```
Для того чтобы писать кавычки внутри кавычек, используется обратный слэш


```suggestion
	name = "устройство \"Хамелеон\""
```

Ну а вообще я думаю было бы достаточно здесь просто хамелиона

```suggestion
	name = "\"Хамелеон\""
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "Вы можете использовать его на шлюзах или АПЦ, чтобы попытаться взломать их, не перерезая проводов."
+
+/obj/item/device/dna_sampler
+	name = "устройство для забора ДНК"
```
Как насчёт...
```suggestion
	name = "устройство забора ДНК"
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "Ручной аварийный фонарь."
+
+/obj/item/device/flashlight/upgraded
+	name = "\improper светодиодный фонарик"
```
С русским языком эти префиксы вроде и не работают
```suggestion
	name = "светодиодный фонарик"
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+
+/obj/item/device/flashlight/lantern
+	name = "лампа"
+	desc = "Лампа для шахтёров."
```
Чисто моё предложение, субъективщина:
```suggestion
	desc = "Шахтёрская лампа."
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+
+/obj/item/device/flashlight/flare
+	name = "флаер"
+	desc = "Красный стандартный сигнальный флаер. На боковой стороне имеется инструкция: 'Потяните за шнур, зажгите'."
```
*Не забыть посмотреть инструкцию в оригинале и сравнить с переводом*

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+
+/obj/item/device/flashlight/lamp/lava
+	name = "лавовая лампа"
+	desc = "Декоративный светильник в стиле 'китчбэк'. Нуарная версия."
```
```suggestion
	desc = "Декоративный светильник в стиле \"китчбэк\". Нуарная версия."
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "Декоративная лава-лампа желтого цвета."
+
+/obj/item/device/geiger
+	name = "счетчик Гейгера"
```
```suggestion
	name = "счётчик Гейгера"
```

Если что, подсказка такая:
Буква <kbd>Ё</kbd> находится слева сверху на клавиатуре, под клавишей <kbd>Esc</kbd>

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+
+/obj/item/device/oxycandle
+	name = "кислородная свеча"
+	desc = "Стальная трубка с надписью 'КИСЛОРОД - ПОТЯНИТЕ ЗА ШНУР, ЧТОБЫ ЗАЖЕЧЬ' на боку. Небольшая этикетка гласит. <span class='warning'>ПРЕДУПРЕЖДЕНИЕ: НЕ ДЛЯ ОСВЕЩЕНИЯ. ВОСПЛАМЕНЯЕТ ГОРЮЧИЕ ГАЗЫ.</span>"
```
`\"`

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	name = "проецируемый предмет"
+
+/obj/item/device/spy_bug
+	name = "жучёк"
```
На этот рез Ё нашёл, но здесь О
```suggestion
	name = "жучок"
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+
+/obj/item/device/suit_sensor_jammer
+	name = "небольшое устройство"
+	desc = "У этого предмета торчат тупые шипы из пластика."
```
Из предмета может торчат...

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "У этого предмета торчат тупые шипы из пластика."
+
+/obj/item/device/t_scanner
+	name = "\improper терагерцовый сканер"
```
```suggestion
	name = "терагерцовый сканер"
```

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "Устройство, которое может записывать на кассеты и воспроизводить их. При воспроизведении он автоматически переводит содержимое."
+
+/obj/item/device/tape
+	name = "магнитная лента"
```
А не кассета?

SuhEugene commented on mods/_rus/code/obj/device/another.dm:
```diff
@@ -0,0 +1,251 @@
+	desc = "Небольшой универсальный клапан с двухголовой жаропрочной трубкой. Этот механизм имеет стандартный размер для соединения с портативными газовыми баллонами."
+
+/obj/item/device/camera/tvcamera
+	name = "Дрон с онлайн камерой"
```
```suggestion
	name = "Дрон с онлайн-камерой"
```

Vuradu committed:
Translate tile & leather

Перевёл шкуры и тайлы(предметом)

Vuradu committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into Translate_pref

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "стандартный ключ шифрования"
+	desc = "Ключ шифрования для радиогарнитуры. Содержит шифрключи."
+
+/obj/item/device/encryptionkey/headset_sec
+	name = "ключ шифрования службы безопасности"
+
+/obj/item/device/encryptionkey/headset_eng
+	name = "ключ шифрования инженера"
+
+/obj/item/device/encryptionkey/headset_rob
+	name = "ключ шифрования робототехника"
+
+/obj/item/device/encryptionkey/headset_med
+	name = "ключ шифрования медика"
+
+/obj/item/device/encryptionkey/headset_sci
+	name = "ключ шифорвания учёного"
+
+/obj/item/device/encryptionkey/headset_medsci
+	name = "ключ шифрования исследовательского медика"
+
+/obj/item/device/encryptionkey/headset_com
+	name = "ключ шифрования командования"
+
+/obj/item/device/encryptionkey/merchant
+	name = "ключ шифрования продавца"
+
+/obj/item/device/encryptionkey/heads/captain
+	name = "ключ шифрования командира"
```
Я бы на вашем месте перевёл бы как и "Стандартный ключ шифрования"
То есть:
- "медицинский ключ шифрования"
- "научный ключ шифрования"
- ...
- "командный ключ шифрования"

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "интегрированный ключ шифрования ИИ"
+	desc = "Интегрированный ключ шифрования."
+
+/obj/item/device/encryptionkey/heads/rd
+	name = "ключ шифрования главного научного сотрудника"
```
А вот это кстати адекватно звучит, но длинно.

Было бы наверное неплохо написать
"ключ шифрования ДИ" и в описание присунуть, что это ключ шифрования директора исследований.

И аналогично с остальными главами.

Vuradu committed:
fix include

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	desc = "Гарнитура для спецназа."
+
+/obj/item/device/radio/intercom
+	name = "интерком (общее)"
```
```suggestion
	name = "интерком (Общий)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "интерком (Кастомный)"
+
+/obj/item/device/radio/intercom/hailing
+	name = "интерком (Hailing)"
```
```suggestion
	name = "интерком (Гостевой)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "\improper интерком специальных операций"
+
+/obj/item/device/radio/intercom/department/medbay
+	name = "интерком (Медбей)"
```
```suggestion
	name = "интерком (Медблок)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "интерком (Служба безопасности)"
+
+/obj/item/device/radio/intercom/entertainment
+	name = "интерком тдела досуга"
```
```suggestion
	name = "интерком (Развлекательный)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "интерком (Приватный)"
+
+/obj/item/device/radio/intercom/specops
+	name = "\improper интерком специальных операций"
```
```suggestion
	name = "интерком  (Спец. операции)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "интерком (Медбей)"
+
+/obj/item/device/radio/intercom/department/security
+	name = "интерком (Служба безопасности)"
```
```suggestion
	name = "интерком (ОЗА)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+
+/obj/item/device/radio/intercom/syndicate
+	name = "незаконный интерком"
+	desc = "Говорить сюда. Злобно."
```
АХАХАХА
Офигенно написано :+1:

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+
+/obj/item/intercom_electronics
+	name = "плата интеркома"
+	desc = "На этикетке написано 'Интерком'. Интересно, для чего он нужен?"
```
```suggestion
	desc = "На этикетке написано \"Интерком\". Интересно, для чего он нужен?"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	desc = "На этикетке написано 'Интерком'. Интересно, для чего он нужен?"
+
+/obj/item/device/radio/intercom/locked/ai_private
+	name = "\improper интерком ИИ"
```
```suggestion
	name = "интерком (ИИ)"
```

SuhEugene commented on mods/_rus/code/obj/device/radio.dm:
```diff
@@ -0,0 +1,250 @@
+	name = "коротковолновое радио"
+
+/obj/item/device/radio/hailing
+	name = "коротковолновое радио (Hailing)"
```
Если речь о канале:
```suggestion
	name = "коротковолновое радио (Гостевой)"
```

Если речь о радио:

```suggestion
	name = "коротковолновое радио (Гостевое)"
```

SuhEugene commented on mods/_rus/code/obj/robot.dm:
```diff
@@ -0,0 +1,86 @@
+	desc = "Вы не должны это видеть!"
+
+/obj/item/borg/upgrade/uncertified/party
+	name = "\improper официальный партийный модуль Madhouse Productions"
```
```suggestion
	name = "официальный партийный модуль Madhouse Productions"
```

SuhEugene commented on mods/_rus/code/obj/robot.dm:
```diff
@@ -0,0 +1,86 @@
+	desc = "Реактивный ранец на углекислом газе, пригодный для горных работ в условиях низкой гравитации."
+
+/obj/item/borg/upgrade/rcd
+	name = "инженерный RCD мудуль для робота"
```
```suggestion
	name = "инженерный RCD модуль для робота"
```

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	desc = "Это сверхпрочный черный полимер. Пора выносить мусор!"
+
+/obj/item/storage/bag/trash/bluespace
+	name = "блюспейс Мусорный мешок"
```
```suggestion
	name = "блюспейс мешок для мусора"
```

```suggestion
	name = "блюспейс мусорный мешок"
```

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	name = "библия"
+	desc = "Неоднократно прикладывать к голове."
+
+/obj/item/storage/bible/bible
+	name = "\improper Библия"
+	desc = "Основной религиозный текст христианства."
+
+/obj/item/storage/bible/tanakh
+	name = "\improper Танах"
+	desc = "Основной религиозный текст иудаизма."
+
+/obj/item/storage/bible/quran
+	name = "\improper Quran"
+	desc = "Основной религиозный текст ислама."
+
+/obj/item/storage/bible/kojiki
+	name = "\improper Kojiki"
+	desc = "Сборник мифов древней Японии."
+
+/obj/item/storage/bible/aqdas
+	name = "\improper Kitab-i-Aqdas"
+	desc = "Основной религиозный текст Веры Бахаи."
+
+/obj/item/storage/bible/guru
+	name = "\improper Гуру Грантх Сахиб"
+	desc = "Основной религиозный текст сикхской веры."
```
Убрать импропер у всего, что не является латиницей

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	desc = "Маленький ланчбокс. На ней изображен симпатичный научный кот из популярного шоу!"
+
+/obj/item/storage/lunchbox/nt
+	name = "\improper ланчбокс марки 'NanoTrasen'"
```
`\"`

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	desc = "Небольшой ланчбокс. Он украшен логотипом CTI!"
+
+/obj/item/storage/lunchbox/nymph
+	name = "\improper ланчбокс с Дионой"
```
```suggestion
	name = "ланчбокс с Дионой"
```

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	name = "коробка для пончиков"
+
+/obj/item/storage/cigpaper
+	name = "\improper Папиросная бумага генерала Эрика"
```
```suggestion
	name = "Папиросная бумага генерала Эрика"
```

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	desc = "Упаковка ароматизированных лечебных леденцов. Идеально подходит для лечения небольших болячек."
+
+/obj/item/storage/mre
+	name = "иРП, Меню 1"
```
```suggestion
	name = "ИРП, Меню 1"
```
Аэээ, ну и ниже так же. И же тоже сокращение от "индивидуальный"...

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	name = "иРП, Меню 8"
+	meal_desc = "Это меню 8, Острый чили."
+
+/obj/item/storage/mre/menu9
+	name = "вегетарианский ИРП"
+	meal_desc = "Это меню 9, Отварной рис (безопасно для Скреллов)."
+
+/obj/item/storage/mre/menu10
+	name = "протеиновый ИРП"
+	meal_desc = "Это меню 10, Протеин."
+
+/obj/item/storage/mre/menu11
+	name = "иРП с мелками"
+	meal_desc = "У этого нет меню. Очень странно..."
```
Могу предложить продолжить тему сверху и бахнуть "ИРП, Вегетарианский", "ИРП, Протеиновый", "ИРП, Мелковый" (или "ИРП, Мелки")

SuhEugene commented on mods/_rus/code/obj/storage/storage.dm:
```diff
@@ -0,0 +1,309 @@
+	desc = "Гладкая, прочная коробка."
+
+/obj/item/storage/backpack/satchel/syndie_kit
+	desc = "Изящная, прочная сумка."
```
Нейм потерялся

Vuradu replied:
нейма нет в оригинале

SuhEugene commented on mods/_rus/code/obj/storage/storage_backpack.dm:
```diff
@@ -0,0 +1,260 @@
+	desc = "Рюкзак, открывающий локализованный блюспейс карман."
+
+/obj/item/storage/backpack/santabag
+	name = "\improper мешок подарков Санты"
```
```suggestion
	name = "мешок подарков Санты"
```

SuhEugene commented on mods/_rus/code/obj/storage/storage_backpack.dm:
```diff
@@ -0,0 +1,260 @@
+	desc = "Пригодится как для переноски дополнительного снаряжения, так и для гордого провозглашения своей преданности выбранному злобному божеству."
+
+/obj/item/storage/backpack/clown
+	name = "\improper Giggles von Honkerton"
```
С латиницей импропер не удалять

SuhEugene reviewed (commented):
Ладно, с меня хватит, дальше лень проверять

Vuradu committed:
Юджин влез

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

Vuradu committed:
Translate circuitboard and aonther staff

Vuradu committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into Translate_pref

Vuradu committed:
Translate implants and materials

Vuradu committed:
Translate more circuitboard and lot of staff

Vuradu committed:
Translate all "weapon" (not guns)

Vuradu committed:
fix include

Vuradu committed:
Translate a lot of staff, toys and another shit

Vuradu committed:
(!!!) Translate trash (!!!)

Перевёл мусор и ещё пару вещей, над мусором ещё нужно работать, переводы только рабочие. (Не ебу за лор, нужны советы)

Vuradu committed:
Translate all "effects" folder

Vuradu committed:
Translate crates and lockers

SuhEugene commented:
Пиздец

Vuradu committed:
Translate areas shuttle and some 4 deck

Vuradu committed:
Translate 4th and 3rd deck

Vuradu committed:
fix include

Vuradu committed:
Translate 2nd deck areas

Vuradu committed:
Translate 1st deck

Vuradu committed:
Final translate areas

translate bridge deck

Vuradu committed:
Try fix shit

Vuradu committed:
finaly fix recursion error

Vuradu committed:
Translate some atmo machines

Vuradu committed:
fix correct work lifts

Удалил датумы из перевода, теперь лифты переведены и работают корректно

Vuradu committed:
Translate all doors and some machines

Vuradu committed:
Translate some doors and machines

Vuradu committed:
Translate machinery and some kitchen staff

Vuradu committed:
Translate vending, some pipes, machines and another staff

Vuradu committed:
Translate and fix

Перевёл машинерию, "все" вендинги и ещё по мелочи. зафиксил инициализацию med pouch

Vuradu committed:
Translate some machines and items

Vuradu committed:
fix translate messenger backpack

Vuradu committed:
fix names medkit

Vuradu committed:
fix name bandage

Vuradu committed:
Translate some machines and items

Vuradu committed:
Translate some items and atmo machines

Vuradu committed:
Translate all augments

Vuradu committed:
Translate some clothes

Vuradu committed:
fix translate

Vuradu committed:
Translate head wear and fix staff

Vuradu committed:
Translane masks and some staff

Vuradu committed:
Translate rigs and modules

Vuradu committed:
Translate void suits

YegorKandziuba committed:
Merge branch 'dev-sierra' into Translate_pref

Vuradu committed:
Translate space suits

Vuradu committed:
Translate armor and some suits

Vuradu committed:
fix translate

Vuradu committed:
Translate misc suits

Vuradu committed:
Translate some "under" cloth

Vuradu committed:
Translate accessories and more cloth

Vuradu committed:
Translate jumpsuits

Vuradu committed:
Translate all "clothing"

code\modules\clothing

Vuradu committed:
Translate some staff

Vuradu committed:
Translate autolath

Vuradu committed:
Translate holodeck and hydropon

Vuradu committed:
Fix translate seeds name

Vuradu committed:
Translate inegrated ciruit and another staff

LapaKotika committed:
sdgge

Vuradu committed:
Revert "sdgge"

This reverts commit f81bd9356a85cdf644c3f2e36d45198aff1e415a.

Vuradu committed:
Translate books

LapaKotika committed:
translate gases and staff

LapaKotika committed:
fix

LapaKotika committed:
Merge branch 'Translate_pref' of https://github.com/Vuradu/SierraBay12 into Translate_pref

LapaKotika committed:
translate gases and staff

Vuradu committed:
Translate "materials"?

LapaKotika committed:
trans xenoarcheology

LapaKotika committed:
Revert "trans xenoarcheology"

This reverts commit 698400b02cfb991c7e52b2cf9c3633b7c2386135.

LapaKotika committed:
Revert "Revert "trans xenoarcheology""

This reverts commit 5c3187ee6f3f84fdf7b2c09bde688d52a0651789.

LapaKotika committed:
add to xeno

LapaKotika committed:
trans musical instruments, lift, magic staff
